pkg/plugins/plugins.go

```diff
@@ -0,0 +1,63 @@
+package plugins
```

Comment:
Can we move this to azure-helm to implement against it?

Reply:
moved